src/components/VirtualThought.tsx

```diff
@@ -99,6 +101,9 @@ const VirtualThought = ({
   const fontSize = useSelector(state => state.fontSize)
   const note = useSelector(state => noteValue(state, thought.id))
   const ref = useRef<HTMLDivElement>(null)
+  const noOtherContexts = useSelector(
+    state => isContextViewActive(state, simplePath) && getContexts(state, thought.value).length <= 1,
+  )
```

Comment:
I think you can select the context view active state and ignore the number of contexts. We probably want to recalculate the height whenever the context view is toggled.

Reply:
Yes, I removed the context length check.